common/src/main/java/org/jboss/hal/testsuite/creaper/command/MessagingUtils.java

```diff
@@ -0,0 +1,62 @@
+package org.jboss.hal.testsuite.creaper.command;
+
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.OnlineManagementClient;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+
+import java.util.List;
+
+final class MessagingUtils {
```

Comment:
What about introduce `org.jboss.hal.testsuite.creaper.command.messaging` and move classes like this to the new package to make `org.jboss.hal.testsuite.creaper.command` better arranged? WDYT?